fast64_internal/f3d/f3d_material.py

```diff
@@ -2460,45 +2460,54 @@ class RDPSettings(bpy.types.PropertyGroup):
         name="Z Buffer",
         default=True,
         update=update_node_values_with_preset,
+        description="Turns on/off Z-Buffer.  Z-Buffer set to 0 if disabled."
     )
     g_shade: bpy.props.BoolProperty(
         name="Shading",
         default=True,
         update=update_node_values_with_preset,
+        description="Turns on/off shading. Shade register set to 0 if disabled."
```

Comment:
Same deal as with Z, I'd suggest something like "Compute and send shade coordinates of tris. Disable if not using vertex colors, lighting, or fog."